scikits/umfpack/umfpack.py

```diff
@@ -505,13 +505,16 @@ def symbolic(self, mtx):
         if self.isReal:
             status, self._symbolic\
                     = self.funs.symbolic(mtx.shape[0], mtx.shape[1],
-                                          mtx.indptr, indx, mtx.data,
+                                          mtx.indptr.astype('int32'),
+                                          indx.astype('int32'),
+                                          mtx.data,
                                           self.control, self.info)
         else:
             real, imag = mtx.data.real.copy(), mtx.data.imag.copy()
             status, self._symbolic\
                     = self.funs.symbolic(mtx.shape[0], mtx.shape[1],
-                                          mtx.indptr, indx,
+                                          mtx.indptr.astype('int32'),
+                                          indx.astype('int32'),
                                           real, imag,
```

Comment:
Does it work on Windows when the `.astype('int32')` conversion is reverted? It does on Linux.

This retyping effectively negates  PR #23, i.e. to solve problems with matrices such that, during factorization, the int32 indices are not enough to store all matrix nonzeros.